packages/flame_network_assets/example/pubspec.yaml

```diff
@@ -5,7 +5,7 @@ publish_to: 'none'
 version: 1.0.0+1
 
 environment:
-  sdk: '>=2.18.0 <4.0.0'
+  sdk: ">=3.0.0 <4.0.0"
```

Comment:
We should add this to `melos.yaml` so that it can be set for all packages at once